charts/victoria-metrics-auth/values.yaml

```diff
@@ -0,0 +1,194 @@
+# Default values for victoria-metrics-auth.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of replicas of vmauth
+replicaCount: 1
+
+image:
+  # -- Victoria Metrics Auth Docker repository and image name
+  repository: victoriametrics/vmauth
+  # -- Tag of Docker image
+  tag: v1.46.0-cluster
+  # -- Pull policy of Docker image
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+containerWorkingDir: "/"
+
+rbac:
+  create: true
+  pspEnabled: true
+  annotations: {}
+  extraLabels: {}
+
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use. If not set and create is true, a name is generated using the fullname template
+  name:
+
+# -- See `kubectl explain poddisruptionbudget.spec` for more. Ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
+  # maxUnavailable: 1
+  labels: {}
+
+# -- WARN: need to specify at least one remote write url
+remoteWriteUrls: []
```

Comment:
vmauth doesnt support remoteWriteUrl

Reply:
Thanks for the info @f41gh7, I will remove this in the next commit.